src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -51,18 +51,21 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
 
             if (options.IsGcEnabled)
             {
-                // TODO: Almost all the ObservableGauge should be ObservableUpDownCounter (except for CPU utilization).
-                // Replace them once ObservableUpDownCounter is available.
-                this.meter.CreateObservableGauge($"{metricPrefix}gc.count", () => GetGarbageCollectionCounts(), description: "GC Count for all generations.");
+                this.meter.CreateObservableCounter($"{metricPrefix}gc.count", () => GetGarbageCollectionCounts(), description: "Number of times garbage collection has occurred since process start.");
 
 #if NETCOREAPP3_1_OR_GREATER
-                this.meter.CreateObservableCounter($"{metricPrefix}gc.allocated.bytes", () => GC.GetTotalAllocatedBytes(), "By", "Allocation Rate.");
+                this.meter.CreateObservableCounter($"{metricPrefix}gc.allocated", () => GC.GetTotalAllocatedBytes(), unit: "By", description: "Count of the bytes allocated since the process start, this does not include any native allocations.");
 #endif
 
 #if NET6_0_OR_GREATER
-                this.meter.CreateObservableGauge($"{metricPrefix}gc.committed", () => GetGarbageCollectionCommittedBytes(), "By", description: "GC Committed Bytes.");
-                this.meter.CreateObservableGauge($"{metricPrefix}gc.heapsize", () => GetGarbageCollectionHeapSizes(), "By", "Heap size for all generations.");
-                this.meter.CreateObservableGauge($"{metricPrefix}gc.fragmentation.size", GetFragmentationSizes, description: "GC fragmentation.");
+                // TODO: change to ObservableUpDownCounter
+                this.meter.CreateObservableGauge($"{metricPrefix}gc.committed", () => GetGarbageCollectionCommittedBytes(), unit: "By", description: "The committed bytes of the managed heap, as observed during the latest garbage collection. If garbage collection has not occurred yet, the value will be unavailable.");
```

Comment:
```suggestion
                this.meter.CreateObservableGauge($"{metricPrefix}gc.committed", () => GetGarbageCollectionCommittedBytes(), unit: "By", description: "The committed bytes of the managed heap, as observed during the latest garbage collection. If a garbage collection has not occurred since process start, the value will be unavailable.");
```

Reply:
Don't know, I personally find the original version better.
@CodeBlanch what's your suggestion?

Reply:
How about...

`The committed bytes of the managed heap as observed during the most recent garbage collection. The value will be unavailable until garbage collection has occurred.`

Reply:
Like it, let me make an update.

Reply:
I like it too